_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+They also uphold the principles of the [ESDC OSS Framework](https://dialogue/grp/CO7397744/MasterLib/ESDC%20Open%20Source%20Management%20Framework%20v%201.2.docx), concentrating on making them actionable.
+
+Below are the six principles that govern this strategy:
+
+### Open First
+
+> Use open standards and OSS first.
+> Work in the open by default.
+
+The GC is an enthusiastic user of and contributor to OSS.
+This is reflected in its Digital Standards and Policy on Service and Digital, which encourages the IT community to tap into the growing potential of open standards and OSS as well as join forces with major commercial players and communities, and mobilise co-creation capacity to support departments in pioneering new solutions.
+
+The GC [Guide for using OSS](https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/open-source-software/guide-for-using-open-source-software.html) provides recommendations how to: actively and fairly consider Open Source Software; verify OSS ownership or licence; evaluate support options; use OSS with or without modification.
+
+### Transform
+
+> We harness the working principles of OSS;
+> we innovate and co-create, share and reuse, and together build user-centric, data-driven public services.
+
+We adopt a working culture based on the principles of open, sharing and reuse.
+We nudge ESDC and its working methods forward by highlighting the crucial role of open standards and OSS and related best practices for co-creation and collaboration.
+Ideally, this transformation will go beyond IT, so that we apply open methods when working together and sharing information.
+
+Open standards and OSS offer practical tools and methods for teams to work together and co-create solutions, to share their expertise freely in order to obtain better results and have their solutions in actual use sooner.
+In these teams, which sometimes involve several directorates-general, developers look at software code written by their colleagues and try to insert new components.
+They combine parts, borrow tools and recipes, and together build and extend a collection of solutions they can use to run digital government solutions.
+
+Inner source (the creation of working cultures based on open principles and share code within the organization) will be promoted, in order to bring teams closer to working in the open and encourage sharing and reuse within the organisation.
+This is a step towards opening up more projects as open source code and sharing them with the outside world.
+
+### Share
+
+> All source code must be released under an appropriate OSS license.
+> We enable incidental contributions to related open standards and OSS projects.
+
+Wherever it makes sense to do so, the ESDC will share the source code of its future IT projects.
+For publication of these projects, the OSS licences recommended by GC (MIT, Apache2, GPL3, AGPL3 or LGPL3) will be preferred.
```

Comment:
Should we state the licenses or just link to the TBS page that has the preferred licenses?  Linking means that we don't have to update our strategy if TBS' recommendations change.